share/github-backup-utils/ghe-backup-es-audit-log

```diff
@@ -26,31 +26,34 @@ else
   es_port=9200
 fi
 
-if ! indices=$(ghe-ssh "$host" "curl -s \"localhost:$es_port/_cat/indices/audit_log*?h=index"\"); then
+if ! indices=$(ghe-ssh "$host" "curl -s \"localhost:$es_port/_cat/indices/audit_log*?h=index,pri.store.size&bytes=b\""); then
   echo "Error: failed to retrieve audit log indices." 1>&2
   exit 1
 fi
 
+# Exit if no indices were found
+[ -z "$indices" ] && exit
+
 # Determine if the audit log migration has occurred or is needed.
 if echo 'set -o pipefail; ! test -e /data/user/common/es-scan-complete && test -f /usr/local/share/enterprise/run-audit-log-transitions.sh' | ghe-ssh "$host" /bin/bash; then
   if echo 'set -o pipefail; echo n | /usr/local/share/enterprise/run-audit-log-transitions.sh > /dev/null 2>&1 && touch /data/user/common/es-scan-complete' | ghe-ssh "$host" /bin/bash; then
     touch $GHE_SNAPSHOT_DIR/es-scan-complete
   fi
 fi
 
-current_index="audit_log(-[0-9]+)?-$(ghe-ssh "$host" 'date +"%Y-%m"')(-[0-9]+)?"
+while read index; do
```

Comment:
Any particular reason for the switch from a `for` loop to a `while`?  Curious as I nearly missed this was reading in `$indices` as it's right at the end.

Reply:
I went with a `while` to avoid the need to switch `IFS` back and forth. I've updated to use a `for` loop now as is it more readable as you mentioned.